src/pages/my_page/user_profile/MyProfile.tsx

```diff
@@ -102,91 +102,103 @@ export const MyProfile = () => {
               <>
                 <Typography>{t('my_page.my_profile.user_profile_description')}</Typography>
                 <Formik initialValues={initialValues} onSubmit={updatePerson} enableReinitialize>
-                  {({ isSubmitting, dirty }: FormikProps<CristinPersonFormData>) => (
-                    <Form>
-                      <Box
-                        sx={{
-                          display: 'grid',
-                          gridTemplateAreas: { xs: '"user-info" "profile-picture"', md: '"user-info profile-picture"' },
-                          gridTemplateColumns: { xs: '1fr', md: '2fr auto' },
-                          rowGap: '0.5rem',
-                        }}>
+                  {({ isSubmitting, dirty, resetForm }: FormikProps<CristinPersonFormData>) => {
+                    return (
+                      <Form>
```

Comment:
ser ingen grunn til denne endringen, kanskje bare en følge av debugging?
```suggestion
                  {({ isSubmitting, dirty, resetForm }: FormikProps<CristinPersonFormData>) => (
                      <Form>
```